### ISSUE TITLE
Error when decoding _status_last when the `reset()` function fails

### ISSUE THREAD

**Issue by chrisbailey4** [adafruit/Adafruit_CircuitPython_MLX90393#25]
I ran into a particularly weird case where my chip was in a bad state.  In my attempt to debug, I noticed that the status being set by the line:
https://github.com/adafruit/Adafruit_CircuitPython_MLX90393/blob/d370d3ec90e09c70e5565200af576037e4f52701/adafruit_mlx90393.py#L389
aka `self._status_last = self._transceive(bytes([_CMD_RT]))`. was slightly wrong and if I attempted to call the `display_status` I would get a type error.

I made a local change to address this:
```
data = self._transceive(bytes([_CMD_RT]))
self._status_last = data[0]
```

I am happy to make a PR to address this minor corner case and support the library if you would like.

**Comment by ladyada** [comment]
yep please submit a PR!

**Comment by caternuson** [comment]
Closing. Should be fixed by #26.
